python/ray/actor.py

```diff
@@ -192,7 +212,8 @@ def __init__(self, modified_class, class_id, max_reconstructions, num_cpus,
         ]
 
         constructor_name = "__init__"
-        if constructor_name not in self.actor_method_names:
+        if not self.is_cross_language and \
+                constructor_name not in self.actor_method_names:
```

Comment:
if (not self.is_cross_language and
                constructor_name not in self.actor_method_names):

Reply:
I think one \ is clearer than ()